### PR TITLE
Moon Phases: Convert to structured answer

### PR DESCRIPTION
**Pull request by mintsoft**
###### Description of changes

Ref: https://github.com/duckduckgo/zeroclickinfo-goodies/issues/2767

---

Instant Answer Page: https://duck.co/ia/view/moon_phases

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @mintsoft

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: t/MoonPhases.t [hunk 1]
```diff
@@ -16,42 +16,26 @@ my $named      = qr/(?:New|Full)$space_plus(?:Moon)/;
 my $phases     = qr/$wax_wane|$quarter|$named/;
 
 my $ascii_answer = re(qr/^The current lunar phase is: $phases$/);
-my $html_answer  = re(qr%^The current lunar phase is: <a href="\?q=$phases">$phases</a>$%);
+
+sub build_test
+{
```
Comment (GuiltyDolphin) [inline t/MoonPhases.t]: @mintsoft Btw, what's with the brace on a newline? Goodies is generally pretty cuddly.

Comment (mintsoft) [inline t/MoonPhases.t]: Allman style brackets; a habit from C

Comment (GuiltyDolphin) [inline t/MoonPhases.t]: @mintsoft Lets cuddle - don't think about that phrase too much... You know what I mean.

**Comment by GuiltyDolphin** [comment]
@mintsoft Thanks! :dolphin:

**Pull request merged**